project.godot

```diff
@@ -864,7 +876,7 @@ debug/crash_message="We're sorry, but the game crashed. Please send us the follo
 %s"
 debug/enable_room_selector=true
 debug/room_selector_room_dir="res://game/rooms"
-ui/game_scene="res://addons/escoria-ui-9verbs/game.tscn"
+ui/game_scene="res://addons/escoria-ui-simplemouse/game.tscn"
```

Comment:
I'm not sure we should commit to having the checked-in project using SimpleMouse instead of 9-verbs.

The rest of the edits to the `project.godot` file look ok, though, especially putting in the Simple Dialog settings.